tests/bsim/bluetooth/audio/src/bap_stream_rx.c

```diff
@@ -35,29 +35,29 @@ void bap_stream_rx_recv_cb(struct bt_bap_stream *stream, const struct bt_iso_rec
 
 	if (test_stream->rx_cnt > 0U && info->ts == test_stream->last_info.ts) {
 		log_stream_rx(stream, info, buf);
-		FAIL("Duplicated timestamp received: %u\n", test_stream->last_info.ts);
+		FAIL("Duplicated timestamp received: %u", test_stream->last_info.ts);
```

Comment:
The `FAIL` macro does not add a CR-LF, so I don't think you want to do this change
We should probably fix this at other places as well